include/LLVMSPIRVLib.h

```diff
@@ -137,6 +137,18 @@ using SpecConstInfoTy = std::pair<uint32_t, uint32_t>;
 bool getSpecConstInfo(std::istream &IS,
                       std::vector<SpecConstInfoTy> &SpecConstInfo);
 
+struct SPIRVHeaderData {
```

Comment:
Would it be better to define this struct somewhere in `lib/SPIRV/libSPIRV` subdir to get access to `SPIRVWord` and `SPIRVWORD_MAX` definitions?

Reply:
I believe it won't work. The closest header by meaning would be SPIRVEntry.h. So we would need to include SPIRVEntry.h here, but this very file serves a purpose of compilation of a driver aka llvm-spirv binary, while SPIRVEntry.h (as well as other files in libSPIRV) describes SPIR-V constructs: instructions, types etc.

Reply:
Maybe, my concern wasn't clear. I think that SPIRV header struct shouldn't be defined in include/LLVM* files, because it has nothing with LLVM infrastructure support. It SPIR-V file specific only.
So, it would be better to be defined in lib/SPIRV/* files.